example/spec/parallel/testAPI/bulkOperationsSuccessSpec.js

```diff
@@ -277,7 +277,7 @@ describe('POST /granules/bulk', () => {
       }
     });
 
-    it('starts a workflow with an execution message referencing the correct queue URL', async () => {
+    xit('starts a workflow with an execution message referencing the correct queue URL', async () => {
```

Comment:
Is this on purpose?

Reply:
For posterity: this test was causing "exception: Error: Not Found" which a reindex did not fix, so we're disabling it for now

Reply:
@dopeters  Many many thanks for posting this in the PR!